addons/web/static/src/scss/bootstrap_review_backend.scss

```diff
@@ -138,7 +138,8 @@ $-o-theme-text-colors: null;
 
 // 1. Define placeholder-selectors (aka pseudo-classes) for both white and
 //    black. We'll then extend placeholder rather than classes to avoid
-//    performance issues during SCSS compilation.
+//    performance issues during SCSS compilation. We also define a class for
```

Comment:
This change is actually not needed.
The comment title (L126) is clear enough, sorry for the headhake :)